content/sources/edirom_source_47dde5ab-b8ff-4004-bfde-b65ea5a9a15e.xml

```diff
@@ -22,99 +22,310 @@
     </meiHead>
     <music>
         <facsimile>
-            <surface n="1" xml:id="surface_52b0a53c-a8e9-48ce-aa4c-4bae46ce1556">
-                <graphic height="4452" target="https://digital.blb-karlsruhe.de/blbihd/i3f/v20/6118288" type="facsimile" width="3588"/>
+            <surface lrx="3588" lry="4452" n="1" ulx="0" uly="0" xml:id="surface_52b0a53c-a8e9-48ce-aa4c-4bae46ce1556">
```

Comment:
I didn't get it: Why does the surface have cooridnates? Should not impact anything, I think.

Reply:
No, shouldn't impact anything, it comes from vertaktoid. Actually it's the same idea as in SVG files the container (here `surface` sets the viewport dimensions, the graphic then is place on/in this viewport.